website/source/docs/commands/remote-config.html.markdown

```diff
@@ -50,6 +50,11 @@ The following backends are supported:
   variables can optionally be provided. Address is assumed to be the
   local agent if not provided.
 
+* Etcd - Stores the state in etcd at a given path.
+  Requires the `path` and `endpoints` variables. The `username` and `password`
+  variables can optionally be provided. `endpoints` is assumed to be a
+  comma-separated list of etcd endpoints.
```

Comment:
Reading these docs made me spot something I didn't catch the first time around, because I tested with only one endpoint: isn't `endpoints` a list of URLs? It seems strange to me to delimit a list of URLs with commas because commas are valid within URLs.

I suppose practically-speaking an etcd endpoint URL is only going to contain a scheme, host and port and so there aren't any opportunities for commas there, but could it instead be a space-separated list of URLs so that e.g. auto-linkification algorithms used on various web forums won't be tempted to consider the whole list of endpoints to be a single, long URL?

For example, Github seems to exhibit the behavior I'm talking about:

-config="endpoints=http://127.0.0.1:1234/,http://127.0.0.1:4321/"

Not a huge deal, but something I could see being confusing to people trying to discuss Terraform.